app/src/main/res/layout/no_internet_listitem.xml

```diff
@@ -0,0 +1,58 @@
+<?xml version="1.0" encoding="utf-8"?>
+<layout
+        xmlns:android="http://schemas.android.com/apk/res/android"
+        xmlns:card_view="http://schemas.android.com/apk/res-auto"
+        xmlns:app="http://schemas.android.com/tools" app:ignore="NamespaceTypo">
+
+    <data>
+        <import type="android.view.View"/>
+        <variable name="article" type="com.androidtitan.culturedapp.model.newyorktimes.Article"/>
+        <variable name="multimedium" type="com.androidtitan.culturedapp.model.newyorktimes.Multimedium" />
+    </data>
+
+    <android.support.v7.widget.CardView
+                                        android:id="@+id/placeCard"
+                                        android:layout_width="match_parent"
+                                        android:layout_height="wrap_content"
+                                        android:layout_marginEnd="@dimen/defaultt"
+                                        android:layout_marginStart="@dimen/defaultt"
+                                        android:layout_marginTop="@dimen/half_default"
+                                        android:layout_marginBottom="@dimen/half_default"
+                                        android:background="@drawable/ripple"
+                                        android:clickable="true"
+                                        android:focusable="true"
+                                        android:foreground="?selectableItemBackground"
+                                        card_view:cardElevation="4dp">
+
+        <RelativeLayout
+                android:id="@+id/rippleForeground"
+                android:layout_width="match_parent"
+                android:layout_height="wrap_content">
+
+            <ImageView
+                    android:id="@+id/binding_article_imageview"
+                    android:layout_width="match_parent"
+                    android:layout_height="500dp"
+                    android:layout_alignParentTop="true"
+                    android:scaleType="centerCrop"
+                    imageUrl="@{multimedium.url}"/>
```

Comment:
Expressions within the layout are written in the attribute properties using the "@{}" syntax.